README.rst

```diff
@@ -64,18 +64,17 @@ point in the future.
 
 Installing IMAPClient
 ---------------------
-IMAPClient is listed on the PyPI (Python Package Index). To install
-via PyPI use the pip or easy_install tools::
+IMAPClient is listed on PyPI and can be installed with pip::
 
     pip install imapclient
 
-    easy_install IMAPClient
-
 More installation methods are described in the documentation.
 
 Documentation
 -------------
-IMAPClient's manual is available at http://imapclient.readthedocs.io/. Release notes can be found at http://imapclient.readthedocs.io/#release-history.
+IMAPClient's manual is available at http://imapclient.readthedocs.io/.
+Release notes can be found at
+http://imapclient.readthedocs.io/#release-history.
```

Comment:
This makes me think the releases notes is not up to date for the 2.0. Maybe in another PR.